docs/classical_control.ipynb

```diff
@@ -0,0 +1,311 @@
+{
```

Comment:
<p>It's not clear to me how this represents <code>a != b</code> instead of <code>a = b</code> . If it's not a typo, is there a more intuitive way to write the inequality as a Sympy expression?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/5403/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>